src/main/java/com/github/arix3767/user/dto/UserDto.java

```diff
@@ -0,0 +1,18 @@
+package com.github.arix3767.user.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDto {
```

Comment:
Utwórz model AddUserRequest, który zawiera:
- email
- password
UserDto powinien zawierać:
- id
- email

Reply:
czy ten model musi być tylko Dto czy utworzyć też encję i odzielne konwertery?

Reply:
chodzi o to, że masz jedna encję, która zawiera wszystkie pola i reprezentuje w bazie danych użytkownika, a do tego masz różne widoki, które wyświetlasz klientom aplikacji, czyli różne klasy dto, które zawierają własny podzastaw danych użytkownika